content/committees/specification/compatibility/index.md

```diff
@@ -0,0 +1,532 @@
+---
+title: "Jakarta EE Compatible Implementations and Compatible Products"
+date: "2021-03-11T00:00:00+00:00"
+---
+
+Jakarta EE is a set of open specifications used for enterprise Java application development. It is a continuation of Java EE technology that gathers the wide community of Java developers interested in advancing the specifications for future cloud native Java enterprise applications. Each specification release includes its respective Technology Compatibility Kit (TCK). You can find a link to the TCK on the specification�s release page, for example see Jakarta EE Platform 9.0 or in the case of individual specification see Jakarta Authentication 2.0.
+The Jakarta EE Working Group enables highlighting all compatible implementations of the individual specifications. In addition, the Jakarta EE Working Group highlights compatible products, which are compatible implementations of either the Jakarta EE Platform or the Jakarta EE Web Profile specifications. 
+The first step in having an implementation be designated as Jakarta EE compatible is ensuring it passes the specification�s respective TCK in order to demonstrate compatibility with their corresponding specifications and is compliant with the Eclipse Foundation Technology Compatibility Kit License for the terms of use. This assumes the implementer has reviewed all compatibility requirements in the respective specification, including those in the TCK documentation, notably those in the TCK User Guide.
+The second step, once the TCK is successfully passed, is letting the world know about it. That is,  
+Let the specification project team know by filing a GitHub issue in order to get their acknowledgement and approval, and 
+Help the Jakarta EE Working Group get your compatible implementation of a specification listed on the specification�s page by creating a PR. In the case of compatible products, implementations of either the Jakarta EE Platform specification and/or the Jakarta EE Web Profile specification will be listed on the Compatible Products page. 
+There are additional steps required, including membership in the Jakarta EE working group. Products listed on the Compatible Products pages are welcome to use Jakarta EE Compatible logo per Jakarta EE Trademark Guidelines.
+
+
+
+
```

Comment:
Delete from here and the rest of the document? 
Don't think we should repeat the entire operations guide here...